src/sentry/discover/endpoints/discover_saved_queries.py

```diff
@@ -19,7 +19,7 @@ class DiscoverSavedQueriesEndpoint(OrganizationEndpoint):
     def has_feature(self, organization, request):
         return features.has(
             "organizations:discover", organization, actor=request.user
-        ) or features.has("organizations:events-v2", organization, actor=request.user)
+        ) or features.has("organizations:discover-basic", organization, actor=request.user)
```

Comment:
@markstory shouldn't be become `organizations:discover-query`?

Reply:
Oh I didn't realize you intended this.

Reply:
It *should* but I wanted to add the discover-query flag usage separately.